src/load.c

```diff
@@ -590,13 +635,18 @@ Stock(w_id)
 	char            s_dist_08[25];
 	char            s_dist_09[25];
 	char            s_dist_10[25];
+	char            s_ytd[25];
```

Comment:
I see, that you fill this array, but I don't see any usages. Please, clarify the change.

Reply:
The same for `s_order_cnt` and `s_remote_cnt`.